WooCommerce/UITestsFoundation/Screens/MyStore/MyStoreScreen.swift

```diff
@@ -10,7 +10,8 @@ public final class MyStoreScreen: ScreenObject {
     public init(app: XCUIApplication = XCUIApplication()) throws {
         try super.init(
             expectedElementGetters: [ { $0.staticTexts["Your WooCommerce Store"] }],
-            app: app
+            app: app,
+            waitTimeout: 60
```

Comment:
Sad to see we need to use such huge timeouts 😞  But what can we do... :xcode: :troll: